impl.hpp

```diff
@@ -700,15 +711,10 @@ bool Client::recv_results_and_logout() noexcept {
       return false;
     }
     if (code == msg_logout) {
-      this->on_result("summary", "summary", summary.dump());
```

Comment:
I am not sure I fully understand why this was removed and why the summary is always printed unconditionally below, but this does not seem to be a blocked for merging. I'd appreciated however if you could explain it to me so I can keep in mind the rationale of this change.

Reply:
I know we talked about it already, but I'm adding a summary here as well for future reference.

This `message` was never meant to be parsed as it's free-form. Since we don't need to parse it, we can do without the "parse first, then print when msg_logout is received" logic and directly print it when it's received from the server. 

I'm probably going to change this when I implement the full summary (https://github.com/measurement-kit/libndt/issues/147) anyway.